cloudtrail/lib/botocore/data/iam/2010-05-08/examples-1.json

```diff
@@ -0,0 +1,1577 @@
+      }
+    ],
+    "CreateAccessKey": [
+      {
+        "input": {
+          "UserName": "Bob"
+        },
+        "output": {
+          "AccessKey": {
+            "AccessKeyId": "AKIAIOSFODNN7EXAMPLE",
+            "CreateDate": "2015-03-09T18:39:23.411Z",
+            "SecretAccessKey": "wJalrXUtnFEMI/K7MDENG/bPxRfiCYzEXAMPLEKEY",
+            "Status": "Active",
+            "UserName": "Bob"
+          }
+        },
+        "comments": {
+          "input": {
+          },
+          "output": {
+          }
+        },
+        "description": "The following command creates an access key (access key ID and secret access key) for the IAM user named Bob.",
+        "id": "1fbb3211-4cf2-41db-8c20-ba58d9f5802d",
+        "title": "To create an access key for an IAM user"
+      }
+    ],
+    "CreateAccountAlias": [
+      {
+        "input": {
+          "AccountAlias": "examplecorp"
+        },
+        "comments": {
+          "input": {
+          },
+          "output": {
+          }
+        },
+        "description": "The following command associates the alias examplecorp to your AWS account.",
+        "id": "5adaf6fb-94fc-4ca2-b825-2fbc2062add1",
+        "title": "To create an account alias"
+      }
+    ],
+    "CreateGroup": [
+      {
+        "input": {
+          "GroupName": "Admins"
+        },
+        "output": {
+          "Group": {
+            "Arn": "arn:aws:iam::123456789012:group/Admins",
+            "CreateDate": "2015-03-09T20:30:24.940Z",
+            "GroupId": "AIDGPMS9RO4H3FEXAMPLE",
+            "GroupName": "Admins",
+            "Path": "/"
+          }
+        },
+        "comments": {
+          "input": {
+          },
+          "output": {
+          }
+        },
+        "description": "The following command creates an IAM group named Admins.",
+        "id": "d5da2a90-5e69-4ef7-8ae8-4c33dc21fd21",
+        "title": "To create an IAM group"
+      }
+    ],
+    "CreateInstanceProfile": [
+      {
+        "input": {
+          "InstanceProfileName": "Webserver"
+        },
+        "output": {
+          "InstanceProfile": {
+            "Arn": "arn:aws:iam::123456789012:instance-profile/Webserver",
+            "CreateDate": "2015-03-09T20:33:19.626Z",
+            "InstanceProfileId": "AIPAJMBYC7DLSPEXAMPLE",
+            "InstanceProfileName": "Webserver",
+            "Path": "/",
+            "Roles": [
+
+            ]
+          }
+        },
+        "comments": {
+          "input": {
+          },
+          "output": {
+          }
+        },
+        "description": "The following command creates an instance profile named Webserver that is ready to have a role attached and then be associated with an EC2 instance.",
+        "id": "5d84e6ae-5921-4e39-8454-10232cd9ff9a",
+        "title": "To create an instance profile"
+      }
+    ],
+    "CreateLoginProfile": [
+      {
+        "input": {
+          "Password": "h]6EszR}vJ*m",
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: Critical

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |

 <!-- 7fb15cbe-ef0a-4beb-83ff-e4dade5320e9 -->